### PR TITLE
fix(hogql): documentation improvements

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

First batch of docs improvements.

- I changed "PostHog SQL" back to HogQL
- I changed "JSON" to "Custom" to make people want to click it less... and to better explain what to expect.
- Still kept "SQL" as the insight type, because it looks better.
- I added lines of text above the "SQL" and "Custom" editors

image

image

image

## How did you test this code?

Visually

### CODE REVIEW

File: frontend/src/scenes/insights/Insight.tsx [hunk 1]
```diff
@@ -75,14 +82,28 @@ export function Insight({ insightId }: InsightSceneProps): JSX.Element {
 
             {isUsingDataExploration ? (
                 <>
+                    {actuallyShowQueryEditor && isQueryBasedInsight && query ? (
+                        <div className="mb-2">
+                            Buttons like
+                            <div className="inline-block align-middle">
+                                <OpenEditorButton query={query} type="secondary" size="small" className="ml-1 mr-1" />
+                            </div>
+                            open any part of PostHog as a new insight. Read the{' '}
```
Comment (yakkomajuri) [inline frontend/src/scenes/insights/Insight.tsx:91]: what is a part of posthog?

**Review by fuziontech — APPROVED** [review]
🚢 it

**Comment by mariusandra** [comment]
Ok, we removed the "custom" option from the menu:
image

 I also changed "part of posthog" to "data table", but that's something we should discuss at the standup anyway.

File: frontend/src/scenes/saved-insights/SavedInsights.tsx [hunk 2]
```diff
@@ -73,49 +73,49 @@ export interface InsightTypeMetadata {
...
-        description: 'Use PostHog SQL to query your data',
+        description: 'Use HogQL to query your data.',
         icon: InsightSQLIcon,
         inMenu: true,
     },
     [InsightType.JSON]: {
-        name: 'JSON',
-        description: 'Build custom insights with our JSON query language',
+        name: 'Custom',
+        description: 'Save any data table as an insight, and edit the JSON directly.',
```
Comment (pauldambra) [inline frontend/src/scenes/saved-insights/SavedInsights.tsx:118]:
```suggestion
        description: 'Save components powered by our JSON query language.',
```

**Pull request merged**